server/neptune/workflows/internal/deploy/revision/queue/deployer.go

```diff
@@ -186,22 +186,7 @@ func (p *Deployer) updateCheckRun(ctx workflow.Context, deployRequest terraformW
 		Summary: summary,
 		Actions: actions,
 	}
-
-	version := workflow.GetVersion(ctx, version.CacheCheckRunSessions, workflow.DefaultVersion, 1)
-
-	var err error
-	if version == workflow.DefaultVersion {
-		err = workflow.ExecuteActivity(ctx, p.Activities.GithubUpdateCheckRun, activities.UpdateCheckRunRequest{
-			Title:   request.Title,
-			State:   request.State,
-			Repo:    request.Repo,
-			Summary: request.Summary,
-			Actions: request.Actions,
-			ID:      deployRequest.CheckRunID,
-		}).Get(ctx, nil)
-	} else {
-		_, err = p.GithubCheckRunCache.CreateOrUpdate(ctx, deployRequest.ID.String(), request)
-	}
+	_, err := p.GithubCheckRunCache.CreateOrUpdate(ctx, deployRequest.ID.String(), request)
```

Comment:
According to the docs
```
// It is recommended to keep the GetVersion() call even if single branch is left:
//  GetVersion(ctx, "fooChange", 2, 2)
//  err = workflow.ExecuteActivity(ctx, baz).Get(ctx, nil)
//
// The reason to keep it is: 1) it ensures that if there is older version execution still running, it will fail here
// and not proceed; 2) if you ever need to make more changes for “fooChange”, for example change activity from baz to qux,
// you just need to update the maxVersion from 2 to 3.
//
// Note that, you only need to preserve the first call to GetVersion() for each changeID. All subsequent call to GetVersion()
// with same changeID are safe to remove. However, if you really want to get rid of the first GetVersion() call as well,
// you can do so, but you need to make sure: 1) all older version executions are completed; 2) you can no longer use “fooChange”
// as changeID. If you ever need to make changes to that same part like change from baz to qux, you would need to use a
// different changeID like “fooChange-fix2”, and start minVersion from DefaultVersion again. The code would looks like:
```
it looks like we should not remove the GetVerison() call unless we can make sure older executions have completed and the changeID is no longer used. Should we just leave the version constant as deprecated so that we don't use the same version in the future if we need to make changes to this part of the workflow?

Reply:
regarding 1), older executions have completed? I checked a while back after this and saw we had 0 in progress executions.  I've removed previous version statements as well by checking for in-progress executions.

Would rather not leave code in that makes this hard to read/understand. We should always clean up where we can.  We can always add another version in if we want 2).